ui/src/app/cohort-search/redux/store/initial.ts

```diff
@@ -9,8 +9,8 @@ export const initialState = fromJS({
   entities: {
     searchRequests: {
       [SR_ID]: {
-        includes: ['include0'],
-        excludes: ['exclude0'],
+        includes: [],
```

Comment:
Since we axed these we should also remove their initial implementations (the contents of `groups` in this file).  We'll need to also check the tests / components to make sure everything still works.

Reply:
Done.